services/gateway/sources/version2/mappings/account.js

```diff
@@ -63,10 +63,11 @@ module.exports = {
 			isBanned: '=,boolean',
 			totalVotesReceived: '=,string',
 			missedBlocks: '=,number',
-			producedBlocks: '=,number',
+			producedBlocks: 'producedBlocks,number',
```

Comment:
Why do we need this change? Both the property names are the same. 
```suggestion
			producedBlocks: '=,number',
```

Reply:
Clarified offline!